### ISSUE TITLE
[Bug]: Quota revert to Default value after changing

### ISSUE THREAD

**Issue by ManuelTurnacker** [nextcloud/server#32254]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hello, 

I Allready created this Issue in Nextcloud-Snap, they closed it and told me that i should post this Issue here.

Want to extend Quota on User but after some minutes it will reset to default Value. 
Only User where i can Change the Quota is the Admin User itself

#### Screenshots
direkt after change
![image](https://user-images.githubusercontent.com/104835360/166450726-2c250960-7cab-4fb0-b049-f08bb5387c73.png)

after ~5min
![image](https://user-images.githubusercontent.com/104835360/166451253-1b886b03-ccdb-4520-a367-f182de39b8e2.png)

#### OS/snapd/snap version
nextcloud  23.0.3snap1  30258  latest/stable  nextcloud✓ 
snap    2.55.3
snapd   2.55.3
series  16
ubuntu  20.04
kernel  5.4.0-109-generic

### Steps to reproduce

1. Login as Admin
2. Go to user management
3. I can Change my own Quota ( User Admin)
4. change the Quota of a User from Standard 5GB to 10 or 15GB or even unlimted. 
5. When User Logs into Nextcloud he sees the new Quota
6.  after ~5min the Quota jumps gack to default Value (5GB) even Changing the default Quota wont change anything.

### Expected behavior

Quota will be saved.

### Installation method

Official SNAP package

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database engine version

MySQL

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Accessibility	1.9.0		
Activity	2.15.0		
Auditing / Logging	1.13.0		
Default encryption module	2.11.0		
Deleted files	1.13.0		
File sharing	1.15.0		
LDAP user and group backend	1.13.1		
Log Reader	2.8.0		
Monitoring	1.13.0		
Nextcloud announcements	1.12.0	
Notifications	2.11.1		
Password policy	1.13.0		
Privacy	1.7.0		
Recommendations	1.2.0		
Right click	1.2.0		
Share by mail	1.13.0		
Theming	1.14.0	
Usage survey	1.11.0		
Versions	1.16.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

**Comment by march42** [comment]
I had the very same issue on both my instances. The quota was set by the LDAP integration. I add a screenshot showing the configuration I'm referring to.
I guessed this was intented and appropriate behaviour, so I fixed it by adding the LDAP attribute nextcloudQuota to the user and setting this to "25 GB".

![2022-05-03 (2)](https://user-images.githubusercontent.com/5712929/166515346-22864d71-e12e-4369-bd7f-6574c2d1c30f.png)
